pyccel/codegen/printing/ccode.py

```diff
@@ -944,6 +946,22 @@ def _print_Import(self, expr):
             source = source.name[-1]
         else:
             source = self._print(source)
+        if expr.target:
+           dtype = expr.target.pop().name
+           if source.startswith('stc'):
+             dtype_macro = dtype.upper()
+             _,container_type = source.split("/")
+             class_type_macro = "SET" if container_type.startswith("hset") else "VEC"
+             import_file,_ = source.split('_')
+             if class_type_macro == 'SET':
+                  additional_defines = ''
+             additional_defines = '' if class_type_macro == 'SET' else "#define i_use_cmp\n"
```

Comment:
`"#define i_use_cmp\n"` is specific to lists, it would be better to make this explicit instead of making it the default value. Otherwise when new containers are added this will break

Reply:
Fixed : 81411c4